src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -84,6 +90,23 @@ private void bindHttpServer(Future<Void> startFuture) {
                         startFuture.fail(httpServerAsyncResult.cause());
                     }
                 });
+
+        vertx.setPeriodic(1000, a -> {
```

Comment:
If the timeout is high then checking for expiry every second seems rather pointless.